.github/workflows/SDK-Suppressions-Label.yaml

```diff
@@ -0,0 +1,99 @@
+name: Manage SDK Breakingchange Suppressions Labels Workflow
+
+on:
+  pull_request:
+    branches:
+      - jacktn/suppression-action 
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+  workflow_dispatch:
+  
+jobs:
+  process-sdk-suppressions-labels:
+    name: breakingchange suppressions label Workflow
+    runs-on: ubuntu-latest
+    outputs:
+      process-sdk-suppressions-output: ${{ steps.run-suppressions-script.outputs.suppressions-result-output }}
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          # Required since "HEAD^" is passed to Get-ChangedFiles
+          fetch-depth: 2
+
+      - name: Get GitHub PullRequest Context
+        uses: actions/github-script@v7
+        id: fetch-pullRequest-context
+        with:
+          script: |
+            const pr = context.payload.pull_request;
+            if (!pr) {
+              throw new Error("This workflow must run in the context of a pull request.");
+            }
+            console.log("This action trigger by ", context.eventName);
+            const prContext = {
+              number: pr.number,
+              title: pr.title,
+              html_url: pr.html_url,
+              base: {
+                owner: pr.base.repo.owner.login,
+                repo: pr.base.repo.name,
+                ref: pr.base.ref
+              },
+              head: {
+                owner: pr.head.repo.owner.login,
+                repo: pr.head.repo.name,
+                ref: pr.head.ref
+              },
+              labels: pr.labels.map(label => label.name)
+            };
+            return JSON.stringify(prContext);
+          result-encoding: string
+
+      - name: Get GitHub PullRequest Changed Files
+        shell: pwsh
+        id: get-changedFiles
+        run: |
+          . eng/scripts/ChangedFiles-Functions.ps1
+          $changedFiles = @(Get-ChangedFiles)
+          echo "PR Changed files: $changedFiles"
+          echo "::set-output name=changedFiles::$changedFiles"
+
+      - name: Setup Node and run `npm ci`
+        uses: ./.github/actions/setup-node-npm-ci
+      
+      - name: Run get suppressions label script
+        id: run-suppressions-script
+        env:
+          GITHUB_PULL_REQUEST_CONTEXT: ${{ steps.fetch-pullRequest-context.outputs.result }}
+          GITHUB_TOKEN: ${{ secrets.SPEC_REPO_TOKEN }}
```

Comment:
Can you turn to use the default token GitHub provides?
https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow

Reply:
use default and set premission